src/reports/components/fast-pass-report.tsx

```diff
@@ -53,11 +59,18 @@ export const FastPassReport = NamedFC<FastPassReportProps>('FastPassReport', pro
                         {...props}
                         cardsViewData={props.results.automatedChecks}
                     />
-                </ResultsContainer>
 
-                <p>Placeholder for "Tab stops" heading component</p>
-                <ResultsContainer {...props}>
-                    <p>Placeholder for results.tabStops rendering</p>
+                    <FastPassResultsTitleSection key={4} title="Tab stops" />
+
+                    <TabStopsFailedInstanceSection
+                        key={5}
+                        deps={{
+                            tabStopRequirementActionMessageCreator: undefined,
+                            tabStopsTestViewController: undefined,
```

Comment:
Curiosity q: how come these are explicitly undefined?

Reply:
Good question! These are for interacting with the tab stops results (such as by removing/editing instances) which we don't support in the report. A follow up PR will need to make some changes here since these undefined values relate to why expanding tab stops results isn't working

Reply:
ohh gotcha, thank you for explaining that! :)